the-blue-alliance-ios/View Elements/Match/MatchViewModel.swift

```diff
@@ -44,6 +47,29 @@ struct MatchViewModel {
             }
             return true
         }()
+        
+        // Set RP things, specifically for 2018
+        // Other years can use a similar pattern
+        if match.year == 2018 {
+            redRPString = {
+                guard var redBreakdown = match.breakdown?["red"] as? [String: Any] else { return "" }
+                var rpString: String = ""
+                rpString += (redBreakdown["autoQuestRankingPoint"] as! Bool ? "•" : "")
```

Comment:
This "•" string specifically can live in the match cell. There's no reason for us to be passing it around - we can just say "add an RP label to this view" and then create a label with a "•" inside the cell.

The only reason we'd want something like this to live in the view model would be if we were changing the RP indicator year-to-year or something.